docs/Project-Configuration/Github-Integrations.md

```diff
@@ -89,6 +89,9 @@ Evergreen offers integration with the GitHub checks API. Users have the option t
 #### Configuration
 
 To add a check run to a task, specify it in the list of tasks in the build variant definition.
+Check runs cannot be defined in the task level and will be ignored if done so. 
+
+Specifying a task tag (like in the example below) will create a check run for all the tasks matching the tag. However, this means if there are more tasks matching that tag than the check run limit, it will fail the check run validation. To use the check run on a few select tasks matching the tag, we recommend separating out the tag into two (e.g. [task_tag] into [task_tag_no_check_run] and [task_task_with_check_run]) to not exceed the limit and still use the tags.
```

Comment:
Can we link to documentation where it says "check run limit"?

Reply:
i'm a bit confused about this comment because the documentation for check run is this haha